src/Umbraco.Web.UI.Client/src/views/propertyeditors/blockgrid/umbblockgridsortable.directive.js

```diff
@@ -0,0 +1,717 @@
+(function () {
+    'use strict';
+
+    function isWithinRect(x, y, rect, modifier) {
+        return (x > rect.left - modifier && x < rect.right + modifier && y > rect.top - modifier && y < rect.bottom + modifier);
+    }
+
+
+    function getParentScrollElement(el, includeSelf) {
+        // skip to window
+        if (!el || !el.getBoundingClientRect) return null;
+        var elem = el;
+        var gotSelf = false;
+    
+        while(elem) {
+            
+            // we don't need to get elem css if it isn't even overflowing in the first place (performance)
+            if (elem.clientWidth < elem.scrollWidth || elem.clientHeight < elem.scrollHeight) {
+                var elemCSS = getComputedStyle(elem);
+
+                if (
+                    elem.clientHeight < elem.scrollHeight && (elemCSS.overflowY == 'auto' || elemCSS.overflowY == 'scroll')
+                    ||
+                    elem.clientWidth < elem.scrollWidth && (elemCSS.overflowX == 'auto' || elemCSS.overflowX == 'scroll')
+                ) {
+                    if (!elem.getBoundingClientRect || elem === document.body) return null;
+                    if (gotSelf || includeSelf) return elem;
+                    gotSelf = true;
+                }
+            }
+
+            if(elem.parentNode === document) {
+                return null;
+            } else if(elem.parentNode instanceof DocumentFragment) {
+                elem = elem.parentNode.host;
+            } else {
+                elem = elem.parentNode;
+            }
+        }
+    
+        return null;
+    }
+ 
+
+
+
+    const DefaultConfig = {
+        compareElementToModel: (el, modelEntry) => modelEntry.contentUdi === el.dataset.elementUdi,
+        querySelectModelToElement: (container, modelEntry) => container.querySelector(`[data-element-udi='${modelEntry.contentUdi}']`),
+        identifier: "UmbBlockGridSorter",
+        containerSelector: "ol", // To find container and to connect with others.
+        ignorerSelector: "a, img, iframe",
+        itemSelector: "li",
+        placeholderClass: "umb-drag-placeholder"
+    }
+
+
+
+    function UmbBlockGridSorter() {
+        
+        function link(scope, element) {
+
+
+            let observer = new MutationObserver(function(mutations) {
+                mutations.forEach(function(mutation) {
+                    mutation.addedNodes.forEach(function(addedNode) {
+                        if (addedNode.matches && addedNode.matches(scope.config.itemSelector)) {
+                            setupItem(addedNode);
+                        }
+                    });
+                    mutation.removedNodes.forEach(function(removedNode) {
+                        if (removedNode.matches && removedNode.matches(scope.config.itemSelector)) {
+                            destroyItem(removedNode);
+                        }
+                    });
+                });
+            });
+
+
+            let vm = {};
```

Comment:
@nielslyngsoe this should be `const` unless the object is replaced later, but it only set/modify its properties.

![image](https://user-images.githubusercontent.com/2919859/203014988-c7811c1f-1ee7-4b10-805f-57edc012c050.png)

Reply:
@bjarnef good one, thanks

Reply:
It's being reassigned to `null` when the component is killed. We've reverted it again :)

Reply:
@iOvergaard shouldn't it initially be set to `null` then? :)